web/src/beta/lib/core/engines/Cesium/Feature/index.tsx

```diff
@@ -112,6 +113,17 @@ export default function Feature({
   const cacheable = !data?.updateInterval && !useTransition;
   const urlMD5 = useMemo(() => (data?.url ? generateIDWithMD5(data.url) : ""), [data?.url]);
 
+  const { requestRender } = useContext();
+  // TODO: Find a way to wait updating the entity
+  useEffect(() => {
+    setTimeout(() => {
+      requestRender?.();
+    }, 300);
+  });
+
+  // Need to invoke requestRender when it's unmounted
+  useEffect(() => () => requestRender?.(), [requestRender]);
```

Comment:
(optional) Can you add a brief reason why or at least a link to where its explained?

Reply:
I couldn't the actual resource, but it's necessary...